edx_proctoring/templates/proctoring/seq_proctored_exam_error.html

```diff
@@ -0,0 +1,6 @@
+{% load i18n %}
+<div class="sequence proctored-exam error">
+    <div class="gated-sequence">
+        {% trans "Your exam has been marked as failed due to an error." %}
```

Comment:
@catong can I get some suggested text here. This is the page which will show if the end user goes back to an exam and either:

a) the end user had closed the SoftwareSecure application window, thus stopping the proctoring recording session
b) is that the SoftwareSecure client had abnormally terminated (i.e. crash, or user restarted computer)

In either case, I believe we will view the exam as invalid. I presume we expect the end-user to contact edX support.

FYI, @pbaruah @griffresch for input/feedback.